django_evaluation/settings/local.py

```diff
@@ -241,6 +242,7 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
 }
 
 DATA_BROWSER_HOST = os.environ.get("FREVA_REST_URL", "http://localhost:7777")
+OIDC_URL = os.environ.get("OIDC_URL", "http://localhost:8080//realms/freva/")
```

Comment:
Could you stick to freva-rest and make this `OIDC_DISCOVERY_URL` ? Let's keep the settings as uniform as possible.